### ISSUE TITLE
[Bug] Monster Hunter Rise Crash

### ISSUE THREAD

**Issue by DarkSpyCyber** [Ryujinx/Ryujinx#5264]
### Description of the issue

old ryujinx can succesfully emulate MH, in latest version crashed.

### Reproduction steps

just play

### Log file

00:03:13.988 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100B0401174312B
00:03:13.991 |S| HLE.OsThread.22 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 299}
00:03:13.991 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100B0401174312B
00:03:13.996 |S| HLE.OsThread.22 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 300}
00:03:13.998 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100B0401174312C
00:03:14.002 |S| HLE.OsThread.22 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 300}
00:03:14.002 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100B0401174312C
00:03:15.757 |S| HLE.OsThread.22 ServicePctl IsFreeCommunicationAvailable: Stubbed. 
00:03:18.528 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:03:21.775 |S| HLE.OsThread.22 ServicePctl IsFreeCommunicationAvailable: Stubbed. 
00:03:26.797 |S| HLE.OsThread.22 ServicePctl IsFreeCommunicationAvailable: Stubbed. 
00:03:31.814 |S| HLE.OsThread.22 ServicePctl IsFreeCommunicationAvailable: Stubbed. 
00:03:36.435 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 191023 bytes, profiled functions: 58949).
00:03:37.286 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:03:38.292 |E| HLE.OsThread.10 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 9142 took more than 1000ms, resuming execution...
00:03:52.508 |E| GUI.RenderLoop Application : Unhandled exception caught: Ryujinx.Graphics.Vulkan.VulkanException: Unexpected API error "ErrorDeviceLost".
   at Ryujinx.Graphics.Vulkan.CommandBufferPool.Return(CommandBufferScoped cbs, ReadOnlySpan`1 waitSemaphores, ReadOnlySpan`1 waitDstStageMask, ReadOnlySpan`1 signalSemaphores) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.Vulkan\CommandBufferPool.cs:line 317
   at Ryujinx.Graphics.Vulkan.CommandBufferPool.Return(CommandBufferScoped cbs) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.Vulkan\CommandBufferPool.cs:line 266
   at Ryujinx.Graphics.Vulkan.PipelineFull.FlushCommandsImpl() in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.Vulkan\PipelineFull.cs:line 219
   at Ryujinx.Graphics.GAL.Multithreading.Commands.SetRenderTargetsCommand.Run(SetRenderTargetsCommand& command, ThreadedRenderer threaded, IRenderer renderer) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.GAL\Multithreading\Commands\SetRenderTargetsCommand.cs:line 21
   at Ryujinx.Graphics.GAL.Multithreading.CommandHelper.<>c.<InitLookup>b__5_1[T](Span`1 memory, ThreadedRenderer threaded, IRenderer renderer) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.GAL\Multithreading\CommandHelper.cs:line 41
   at Ryujinx.Graphics.GAL.Multithreading.ThreadedRenderer.RenderLoop()
   at Ryujinx.Graphics.GAL.Multithreading.ThreadedRenderer.RunLoop(ThreadStart gpuLoop) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Graphics.GAL\Multithreading\ThreadedRenderer.cs:line 113
   at Ryujinx.Ui.RendererWidgetBase.Render() in D:\a\Ryujinx\Ryujinx\src\Ryujinx\Ui\RendererWidgetBase.cs:line 449

### OS

windows 10

### Ryujinx version

1.1.869

### Game version

15.0.1

### GPU

nvidia gtx 850m

### RAM

8gb

**Comment by gdkchan** [comment]
You should upload the full log. It's not even possible to see the version of the game with this log snippet.

**Comment by DarkSpyCyber** [comment]
> You should upload the full log. It's not even possible to see the version of the game with this log snippet.

can not upload, 65535 bytes limited

**Comment by DarkSpyCyber** [comment]
> Unless something has recently changed, I don't believe you can play 'Sunbreak' title update versions (e.g. 15.0.1) outside of ldn build <= 1.1.0-ldn3.0 (will always crash at title screen).

no, its crashed after loading, the voice cames out, then crash

**Comment by gdkchan** [comment]
What's the game version?

**Comment by DarkSpyCyber** [comment]
> 

Game version
15.0.1

**Comment by DarkSpyCyber** [comment]
the log can not fully uploaded, and i uploaded to the tem file website: 
https://tmpfiles.org/dl/1522640/ryujinx_1.1.872_2023-06-09_15-46-29.rar
expired at 2023-06-09 08:53 UTC
thanks guys, love your work, salute

**Comment by yogithesymbian** [comment]
this morning ... i play on V1.1.869 and its playable for me ( 13" Macbook Pro M1 ( 8GB )) 
now i have manual download and going to test . 
why manual download cause when trying update from apps from **V1.1.868**  to **V1.1.883** when alr completed downloading its crash the emulator and when re open its still alert need to update **V1.1.883** thats mean my emulator doesnt upate properly . this just fyi and not relate but still relate .

**Comment by dantezero0403** [comment]
[Ryujinx_1.1.896_2023-06-16_07-46-17.log](https://github.com/Ryujinx/Ryujinx/files/11763815/Ryujinx_1.1.896_2023-06-16_07-46-17.log)
I got the same error

**Comment by gdkchan** [comment]
Sounds like a duplicate of #5235. Any version since 10.0.0 does not work.
